### ISSUE TITLE
[Bug]Add-ons don't recognize the local language.

### ISSUE THREAD

**Issue by lomomcat** [mozilla-mobile/fenix#12530]
## Steps to reproduce
Open the options of any add-on or press any add-on in "three points".
### Expected behavior
If a add-on has l10n, it will show the local language.
### Actual behavior
It only showed English.
Notice: **uBlock Origin** has Chinese (Simplified) l10n!
![Screenshot_2020-07-14-10-44-28-839_org.mozilla.fenix.jpg](https://user-images.githubusercontent.com/62339707/87378646-d53a9380-c5c0-11ea-951b-90be7b402877.jpg)
![Screenshot_2020-07-14-10-44-44-603_org.mozilla.fenix.jpg](https://user-images.githubusercontent.com/62339707/87378665-e1265580-c5c0-11ea-8ef2-3df9474a58b0.jpg)
![Screenshot_2020-07-14-10-44-17-241_org.mozilla.fenix.jpg](https://user-images.githubusercontent.com/62339707/87378675-e71c3680-c5c0-11ea-869d-003ca74e2519.jpg)
### Device information

* Android device: Xiaomi Redmi Note 8 Pro
* Fenix version: Nightly 200713 06:47
* Android Version: 10 (MIUI 12, latest patch)
* SoC: MediaTek Helio G90T (ARM64, ARMv8.2-A)

**Comment by lomomcat** [comment]
The problem also appeared on the browser's own pages such as _about:config_.

**Comment by eliserichards** [comment]
@agi90 do you have any thoughts on this?

**Comment by Amejia481** [comment]
Does changing to other locales work? Does the add-on support this locale? I know we have a similar issue but it was for changing system locales https://github.com/mozilla-mobile/fenix/issues/8188#issuecomment-588300299

**Comment by agi90** [comment]
It's broken for Italian too so I don't think this is a locale specific problem. I tried uninstalling/installing the extension and it didn't fix the problem. It's probably related to switching languages, if I install uBlock origin on a clean profile, the localization is correct (I see Italian strings).

**Comment by Amejia481** [comment]
Then it looks like it's related to https://github.com/mozilla-mobile/fenix/issues/8188#issuecomment-588300299

**Comment by agi90** [comment]
Added a comment here https://bugzilla.mozilla.org/show_bug.cgi?id=1615680#c11

**Comment by sflorean** [comment]
> Then it looks like it's related to [#8188 (comment)](https://github.com/mozilla-mobile/fenix/issues/8188#issuecomment-588300299)

Based on the quoted comment, I will close this issue as a dupe of #8188.

**Comment by lomomcat** [comment]
> > Then it looks like it's related to [#8188 (comment)](https://github.com/mozilla-mobile/fenix/issues/8188#issuecomment-588300299)
> 
> Based on the quoted comment, I will close this issue as a dupe of #8188.

@sflorean No! It's not the problem about l10n of add-ons. It's the problem of browser!
Up to now, the problem has not been solved in the latest Nightly.

**Comment by lomomcat** [comment]
**Fenix ​​79** did not have this problem, but **Fenix ​​80** did!

**Comment by lomomcat** [comment]
For another example,

_about:config_ has l10n in **Fenix Beta** (79.0.0-beta.9),too.
![Screenshot_2020-07-23-16-49-48-333_org.mozilla.firefox_beta.jpg](https://user-images.githubusercontent.com/62339707/88268268-c2cd0200-cd04-11ea-9034-26ff49d0fb77.jpg)
But turning to **Fenix Nightly** (Nightly 200723 06:01), it disappeared.
![Screenshot_2020-07-23-16-55-49-816_org.mozilla.fenix.jpg](https://user-images.githubusercontent.com/62339707/88268653-60c0cc80-cd05-11ea-8358-72411e57e022.jpg)

**Comment by Amejia481** [comment]
Let's investigate and see why this issue is happening.

**Comment by Amejia481** [comment]
Thanks @lomomcat this is a regression the web content is not getting translated on nightly for the moment we are discussing the change on bugzilla https://bugzilla.mozilla.org/show_bug.cgi?id=1615680#c14

**Comment by Amejia481** [comment]
[The bug #1615680](https://bugzilla.mozilla.org/show_bug.cgi?id=1615680#c14) it's not related we opened a new   https://bugzilla.mozilla.org/show_bug.cgi?id=1656515

**Comment by sflorean** [comment]
@lomomcat sorry for this misunderstanding with the bugs. 

@Amejia481 we were able to reproduce this bug on Nightly 8/5 but not on Beta 80.0.0-beta.2, and seems that builds affected/not affected were found. Should QA investigate more/something else? Thank you!

**Comment by Amejia481** [comment]
@sflorean Thanks for that info, we are tracking the bug on https://bugzilla.mozilla.org/show_bug.cgi?id=1656515 it will be helpful if we can include that info there too. We are trying to figure out what's causing the issue if QA can help it will be wonderful :)

**Comment by lomomcat** [comment]
@Amejia481 
I found that it has been fixed in **200820 03:27**. 
You can close this issue if you think there is no problem here.
Thanks for all of you!

**Comment by Amejia481** [comment]
Thanks @lomomcat, the gv team recently land a patch. Let's ask to the QA team verify before close :)
